README.md

```diff
@@ -72,31 +72,46 @@ $ snaptel plugin load snap-plugin-publisher-file
 See available metrics for your system:
 ```
 $ snaptel metric list
-NAMESPACE                                              VERSIONS
-/intel/procfs/processes/running                        7
-/intel/procfs/processes/sleeping                       7
-/intel/procfs/processes/waiting                        7
-/intel/procfs/processes/zombie                         7
-/intel/procfs/processes/stopped                        7
-/intel/procfs/processes/tracing                        7
-/intel/procfs/processes/dead                           7
-/intel/procfs/processes/wakekill                       7
-/intel/procfs/processes/waking                         7
-/intel/procfs/processes/parked                         7
-/intel/procfs/processes/*/ps_vm                        7
-/intel/procfs/processes/*/ps_rss                       7
-/intel/procfs/processes/*/ps_data                      7
-/intel/procfs/processes/*/ps_code                      7
-/intel/procfs/processes/*/ps_stacksize                 7
-/intel/procfs/processes/*/ps_cputime_user              7
-/intel/procfs/processes/*/ps_cputime_system            7
-/intel/procfs/processes/*/ps_pagefaults_min            7
-/intel/procfs/processes/*/ps_pagefaults_maj            7
-/intel/procfs/processes/*/ps_disk_ops_syscr            7
-/intel/procfs/processes/*/ps_disk_ops_syscw            7
-/intel/procfs/processes/*/ps_disk_octets_rchar         7
-/intel/procfs/processes/*/ps_disk_octets_wchar         7
-/intel/procfs/processes/*/ps_count                     7
+NAMESPACE 							                                    VERSIONS
```

Comment:
I think that mentioning about possible using `snaptel metric list --verbose`  to see more details would be enough